tests-ng/util/install_tofu.sh

```diff
@@ -1,9 +1,34 @@
 #!/usr/bin/env bash
 
+function die {
+    # Perl like die() function.
+    # https://billauer.co.il/blog/2024/08/bash-exit-with-error/
+    # shellcheck disable=SC2086
+    echo $1
+    exit 1
+}
+
+
 install_tofu() {
 	set -eufo pipefail
 	local tf_dir="${1}"
 
+    # Testing that the necesary biniaries are present at set indpendent of the path.
+    case "$(uname -o)" in
+        Darwin)
+            FIND="/opt/homebrew/bin/gfind"
+            RETRY="/opt/homebrew/bin/retry"
+            test -x $FIND || die "Can't find find. Please install it with 'brew install findutils'"
+            test -x $RETRY  || die "Can't find find. Please install it with 'brew install findutils'"
+            ;;
+        GNU/Linux)
+            FIND="/usr/bin/find"
+            RETRY="/usr/bin/retry"
```

Comment:
Binaries might not always be in `/usr/bin/` and e.g. might break some development setups.
Please use:

```
command -v find
command -v retry
```